_includes/body-landing.html

```diff
@@ -129,52 +123,18 @@ <h5 class="title">How do I?</h5>
             </div>
           </div>
           <div class="row">
-            <div class="col-xs-12 col-md-6">
-              <a href="/engine/reference/builder/">Write a Dockerfile</a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/network/">Manage container networking</a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/compose/compose-file/">Write a Docker Compose file</a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/storage/">Work with volumes and bind mounts</a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/get-started/part3/">Share my image on Docker Hub</a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/config/daemon/">Configure the Docker daemon</a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/get-started/part2/">Build and run an image</a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/config/labels-custom-metadata/">
-                Manage Docker objects
-              </a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/develop/develop-images/multistage-build/">
-                Use multi-stage builds
-              </a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/get-started/kube-deploy/">
-                Scale apps using Kubernetes
-              </a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/engine/reference/commandline/run/">
-                Run a command in a container
-              </a>
-            </div>
-            <div class="col-xs-12 col-md-6">
-              <a href="/get-started/swarm-deploy/">
-                Scale apps using Swarm
-              </a>
-            </div>
+            <div class="col-xs-12 col-md-6"><a href="/engine/reference/builder/">Write a Dockerfile</a></div>
+            <div class="col-xs-12 col-md-6"><a href="/network/">Manage container networking</a></div>
+            <div class="col-xs-12 col-md-6"><a href="/compose/compose-file/">Write a Docker Compose file</a></div>
+            <div class="col-xs-12 col-md-6"><a href="/storage/">Work with volumes and bind mounts</a></div>
```

Comment:
I thought having a link per-line was easier to use; even though some lines are now a bit long, it gives an easier overview how many links are in a section, and to add/remove a link is now just removing a line (or copying one), without having to find where a link block "starts" and "ends", so less chances on making mistakes

Reply:
Thanks. Yes, this makes sense.